Fix `firebase/database` package

Related to https://github.com/zeit/now/issues/1830

## Steps to reproduce

`yarn add firebase`

```js
const firebase = require('firebase/app');
require('firebase/database');
console.log('firebase type is ' + typeof firebase);
```

## Error

```
Cannot find module 'request' loaded from node_modules/@firebase/database/dist/index.node.cjs.js
```

--- a/test/integration/firebase.js
+++ b/test/integration/firebase.js
@@ -1,6 +1,6 @@
 const firebase = require('firebase/app')
 require('firebase/firestore')
-//require('firebase/database')
+require('firebase/database')
 
 firebase.initializeApp({ projectId: 'noop' })
 const store = firebase.firestore()

--- a/test/integration/firebase.js
+++ b/test/integration/firebase.js
@@ -1,5 +1,6 @@
 const firebase = require('firebase/app')
 require('firebase/firestore')
+//require('firebase/database')
 
 firebase.initializeApp({ projectId: 'noop' })
 const store = firebase.firestore()

